share/completions/minikube.fish

```diff
@@ -0,0 +1,79 @@
+function __minikube_no_command
+    set -l cmd (commandline -poc)
+    set -e cmd[1]
+    if test (count $cmd) -eq 0
+        return 0
+    end
+    return 1
+end
+
+function __minikube_using_command
+    set cmd (commandline -poc)
+    if test (count $cmd) -gt 1
+        if test $argv[1] = $cmd[2]
+            return 0
+        end
+    end
+    return 1
+end
+
+complete -c minikube -f -n "__minikube_no_command" -a addons -d "Modify minikube's kubernetes addons"
+complete -c minikube -f -n "__minikube_no_command" -a completion -d "Outputs minikube shell completion for the given shell (bash)"
+complete -c minikube -f -n "__minikube_no_command" -a config -d "Modify minikube config"
+complete -c minikube -f -n "__minikube_no_command" -a dashboard -d "Opens/displays the kubernetes dashboard URL for your local cluster"
+complete -c minikube -f -n "__minikube_no_command" -a delete -d "Deletes a local kubernetes cluster."
+complete -c minikube -f -n "__minikube_no_command" -a docker-env -d "sets up docker env variables; similar to '\$(docker-machine env)'"
```

Comment:
The semicolon is superfluous here - "Set up docker env variables similar to `docker-machine env`".